src/EnergyPlus/HybridEvapCoolingModel.hh

```diff
@@ -0,0 +1,414 @@
+			bool ParseMode(int ModeCounter, std::vector<CMode>* OperatingModes, Real64 correction, Array1D_string Alphas, Array1D_string cAlphaFields, Array1D< Real64 > Numbers, Array1D_string cNumericFields, Array1D<bool>  lAlphaBlanks, std::string cCurrentModuleObject);
+			void InitializeCurve( int curveType, int CurveID);
+			Real64 CalculateCurveVal( Real64 X_1, Real64 X_2, Real64 X_3, Real64 X_4, Real64 X_5, Real64 X_6, int curve_ID);
+			bool InitializeOSAFConstraints(Real64 minOSAF, Real64 maxOSAF);
+			bool InitializeMsaRatioConstraints(Real64 minMsa, Real64 maxMsa);
+			bool InitializeOutsideAirTemperatureConstraints(Real64 min, Real64 max);
+			bool InitializeOutsideAirHumidityRatioConstraints(Real64 min, Real64 max);
+			bool InitializeOutsideAirRelativeHumidityConstraints(Real64 min, Real64 max);
+			bool InitializeReturnAirTemperatureConstraints(Real64 min, Real64 max);
+			bool InitializeReturnAirHumidityRatioConstraints(Real64 min, Real64 max);
+			bool InitializeReturnAirRelativeHumidityConstraints(Real64 min, Real64 max); 
+			bool GenerateSolutionSpace(Real64 ResolutionMsa, Real64 ResolutionOSA);
+			bool MeetsOAEnvConstraints(Real64 Tosa, Real64 Wosa, Real64 RHos);
+		private:
+		
+		};
+		
+		class CSetting
+		{
+		public:
+			CSetting() :Runtime_Fraction(0), Mode(0), Outside_Air_Fraction(0), Unscaled_Supply_Air_Mass_Flow_Rate(0), ScaledSupply_Air_Mass_Flow_Rate(0), Supply_Air_Ventilation_Volume(0), ScaledSupply_Air_Ventilation_Volume(0), Supply_Air_Mass_Flow_Rate_Ratio(0),
+				SupplyAirTemperature(0), Mixed_Air_Temperature(0), SupplyAirW(0), Mixed_Air_W(0), TotalSystem(0), SensibleSystem(0), LatentSystem(0), 
+				TotalZone(0), SensibleZone(0), LatentZone(0), ElectricalPower(IMPLAUSIBLE_POWER), SupplyFanElectricPower(0), SecondaryFuelConsumptionRate(0), ThirdFuelConsumptionRate(0), WaterConsumptionRate(0), ExternalStaticPressure(0), RawW(0){}
+			Real64 Runtime_Fraction;															  
+			Real64 Mode;																		  
+			Real64 Outside_Air_Fraction;														  
+			Real64 Unscaled_Supply_Air_Mass_Flow_Rate;													  
+			Real64 ScaledSupply_Air_Mass_Flow_Rate;
+			Real64 Supply_Air_Ventilation_Volume;
+			Real64 ScaledSupply_Air_Ventilation_Volume;
+			Real64 Supply_Air_Mass_Flow_Rate_Ratio;
+			Real64 SupplyAirTemperature;
+			Real64 Mixed_Air_Temperature;
+			Real64 SupplyAirW;
+			Real64 Mixed_Air_W;
+			Real64 TotalSystem;
+			Real64 SensibleSystem;
+			Real64 LatentSystem;
+			Real64 TotalZone;	// kW
+			Real64 SensibleZone;	 // kW
+			Real64 LatentZone;	// kW
+			Real64 ElectricalPower;
+			Real64 SupplyFanElectricPower;	   
+			Real64 SecondaryFuelConsumptionRate;	
+			Real64 ThirdFuelConsumptionRate;	
+			Real64 WaterConsumptionRate;	   
+			Real64 ExternalStaticPressure;	   
+			//Debug values
+			Real64 RawW;
+
+			// add other fuels, or change name to be fuel
+			CMode oMode;
+			CSetting& operator=(CSetting other)

```

Comment:
What's the gain over the default copy operator here?  What makes a custom one necessary?

Reply:
it used to have something in it that required it, now it doesn't so I removed it. Thanks